lib/tasks/strongmind_defaults.rake

```diff
@@ -16,6 +16,13 @@ namespace :strongmind_defaults do
     puts "Api throttle HWM set to: #{maximum}"
   end
 
+  desc "set default gradebook many_submissions_chunk_size"
+  task :set_gradebook_submissions_chunk_size => :environment do
+    Setting.set('gradebook2.many_submissions_chunk_size', '5')
+    chunk_size = Setting.get('gradebook2.many_submissions_chunk_size', 0)
```

Comment:
Should this be `Setting.set` instead of `.get`? Or potentially `get_or_set`?

Reply:
No, setting.get is to check that read the setting back after setting

Reply:
Ah, the line above also looked like a get when I read it 👍